reactive/rerunner.go

```diff
@@ -103,6 +103,15 @@ func (c *cache) cleanInvalidated() {
 	}
 }
 
+func (c *cache) cleanAll() {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+
+	for key := range c.computations {
+		delete(c.computations, key)
```

Comment:
do we need to do any releasing here?

Reply:
Yeah, in order to be sure we need to.

I took a closer look and it looks like we do already release the node in a goroutine (which should get cleared in the clearInvalidated section), but there's no guarantee that the goroutine is run before we start our next execution run.  So I'm going to keep the explicit deletion here which should be safer.